### ISSUE TITLE
Can I use Amazon Corretto to open Usagi?

### ISSUE THREAD

**Issue by emmaleof** [OHDSI/Usagi#151]
Hello,
I contacted my IT as I was still having issues with downloading Java and loading a vocabulary into Usagi (Ticket: Download Usagi issues  #149). I was informed that Java is no longer supported at our institution due to security concerns with the application, so I cannot download it on my work computer. They mentioned that I might be able to use Amazon Corretto to open Usagi. Do you know if this is possible? If so, what version of Amazon Corretto would I need? Any information would be great.
Thank you.

**Comment by MaximMoinat** [comment]
Unfortunatley I am not familiar with Amazon Corretto. If it allows running Java archives (jar), and has a user interface, then it should be possible.

I am closing this issue. If you want input form others in the OHDSI community with using Amazon Corretto, I would advice to post your question on the OHDSI forums (forums.ohdsi.org).

**Comment by emmaleof** [comment]
Thank you for the suggestion to look at the OHDSI forums (forums.ohdsi.org). I found a related post (https://forums.ohdsi.org/t/does-usagi-require-oracle-java-1-8-can-i-use-openjdk-or-amazon-corretto/13171) and was able to use those instructions. Since I was unable to download Java Oracle on my work computer as the application is no longer supported by our institution, I downloaded Amazon Corretto, which is an approved application. Then, in Windows PowerShell, I used the command line: java -jar C:\Users\username\Downloads\Usagi_v1.4.3.jar to open Usagi.
